tutorials/sac-intro-2-trial/sac-intro-2-trial.md

```diff
@@ -0,0 +1,71 @@
+---
+author_name: Megan Hoy
+author_profile: https://github.com/meganhoy
+title: Set up your SAP Analytics Cloud Trial Account
+description: Sign up for a free trial SAP Analytics Cloud Trial account
+auto_validation: true
+time: 5
+tags: [ tutorial>beginner, topic>cloud]
+primary_tag: software-product>sap-analytics-cloud
+---
+
+<!-- ## Prerequisites
+ - Prerequisite 1
+ - Prerequisite 2 -->
+
+## Details
+### You will learn
+  - What is included in the trial
+  - How to register an SAP.com account
+  - How to register for an SAP Analytics Cloud Trial account
+
+<!-- Add additional information: Background information, longer prerequisites -->
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](SAP Analytics Cloud Trial Overview)]
+
+The free trial of SAP Analytics Cloud gives you a single-user license for SAP Analytics Cloud, with access to business intelligence, enterprise planning, and augmented analytics features. You can connect your data sources, model your data, create stories, try augmented analytics, and do predictive planning. Click [here](https://community.sap.com/topics/cloud-analytics/faq#free-trial) to learn more.
+
+The trial length is 30 days but can be extended for an additional 60 days after the first few weeks of activation.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Register for an SAP.com Account)]
```

Comment:
Please move this to the prerequisites section (remove the image).

Reply:
Our tutorials are tailored to to those looking to Get Started with SAC and so having an SAP account would not be a prerequisite. Instead, we want to show them how to get an account, trial, etc.